site/src/pages/contact.page.tsx

```diff
@@ -180,7 +180,9 @@ const Partners: NextPage = () => {
                 rows={8}
               />
             </div>
-
+            <div hidden>
+              <input type="hidden" name="tags" value="1830797" />
+            </div>
```

Comment:
I used the mailchimp "Embedded forms" creator to figure out how they create HTML forms that attach tags to users that submit them, and this is how.

I've tested this and it seems to work as intended (attaches the tag to anyone submitting the form, whether they are already subscribed or not).